sbin/submitty_autograding_shipper.py

```diff
@@ -115,60 +118,73 @@ def update_worker_json(name, entry):
         host = autograding_worker_to_ship[name]['address']
     except Exception as e:
         print("ERROR: autograding_workers.json entry for {0} is malformatted. {1}".format(e, name))
-        autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID, message="ERROR: autograding_workers.json entry for {0} is malformed. {1}".format(e, name))
-        autograding_utils.log_stack_trace(AUTOGRADING_STACKTRACE_PATH, job_id=JOB_ID, trace=traceback.format_exc())
+        autograding_utils.log_message(AUTOGRADING_LOG_PATH, JOB_ID,
```

Comment:
A copied comment from @MasterOdin from his review of pr #5560 which I agree with: 
> I find for Python (and other languages), that if the list of arguments to a function or whatever extends past one line, to just have the starting element on its own newline with one level of indentation. It's I think a bit nicer looking, easier to add to the list a longer argument (as you're not starting a line at character 78 or whatever) and that indentation is a regular multiple instead of starting at some non-standard number.